exercises/rational-numbers/canonical-data.json

```diff
@@ -400,6 +400,28 @@
             "r": [0, 1]
           },
           "expected": 1.0
+        },
+        {
+          "uuid": "da137f1c-fc94-4035-9813-94f5152a1102",
+          "description": "Raise a negative real number to a rational number with an even numerator",
+          "property": "expreal",
+          "input": {
+            "x": -1,
+            "r": [2, 2]
+          },
+          "expected": 1.0
+        },
+        {
+          "uuid": "f3734f78-91b2-4e59-905e-8c61dbf34f39",
+          "description": "Fails with a negative real number if the numerator of the rational number is odd",
```

Comment:
This implies that an odd numerator is the cause of the error, but it's not the case, the case you introduce on top is equivalent to `"x": -1, "r": [1, 1]` and that is fine. Also `"x": -1, "r": [1, 3]` is `-1`.

I think the criterion is that all powers of a negative number will be complex if the **denominator** of the **reduced** fraction is **even**. Like `"r": [1, 2]`, `"r": [5, 6]`...

Reply:
@jiegillet I revised the description here and the input number for both test cases (I assumed we want to expect the solution to reduce the number before doing the calculation).

Sorry for all my confusion with the concept here, I should've asked for your assistance in defining the test cases. It also didn't help that I missed reducing the number in my gleam implementation. :(

Reply:
An implementation may or may not reducing the input, but in a test we want to test only one thing at a time and bbe crystal clear about it, so I suggest that the input should already be reduced. 

You could add another test that specifically checks that giving an un-reduced number also works.